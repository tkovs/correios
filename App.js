import React from 'react';
import {SafeAreaView, View, Text, StyleSheet} from 'react-native';

const App = () => (
  <SafeAreaView style={styles.container}>
    <View style={styles.body}>
      <Text style={styles.text}>Tracking</Text>
    </View>
  </SafeAreaView>
);

const styles = StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: 'center',
  },
  body: {
    backgroundColor: '#111',
  },
  text: {
    color: '#ddd',
    textAlign: 'center',
  },
});

export default App;
