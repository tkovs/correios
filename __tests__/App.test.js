import 'react-native'
import React from 'react'
import { render } from '@testing-library/react-native'

import App from '../App'

it('renders correctly', () => {
  const container = render(<App />)
  expect(container).toMatchSnapshot()
  expect(1).toEqual(1)
})
