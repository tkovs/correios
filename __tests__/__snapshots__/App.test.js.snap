// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    collapsable={true}
    pointerEvents="box-none"
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <SafeAreaView
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <SafeAreaView
        style={
          Array [
            Object {
              "backgroundColor": "#01417F",
              "elevation": 4,
              "zIndex": 0,
            },
            Object {
              "shadowColor": "#000000",
              "shadowOffset": Object {
                "height": 3,
                "width": 0,
              },
              "shadowOpacity": 0.24,
              "shadowRadius": 4,
            },
          ]
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#01417F",
              "elevation": 0,
              "flexDirection": "row",
              "height": 56,
              "marginTop": 0,
              "paddingHorizontal": 4,
            }
          }
        >
          <View
            style={
              Object {
                "width": 48,
              }
            }
          />
          <TouchableWithoutFeedback>
            <View
              style={
                Array [
                  Object {
                    "flex": 1,
                    "paddingHorizontal": 12,
                  },
                  Array [
                    false,
                    Object {
                      "alignItems": "center",
                    },
                    undefined,
                  ],
                ]
              }
            >
              <Text
                accessibilityRole="header"
                accessibilityTraits="header"
                accessible={true}
                numberOfLines={1}
                style={
                  Array [
                    Object {
                      "color": "#000000",
                      "fontFamily": "System",
                      "fontWeight": "400",
                      "textAlign": "left",
                    },
                    Array [
                      Object {
                        "color": "#ffffff",
                        "fontFamily": "System",
                        "fontWeight": "400",
                      },
                      Object {
                        "fontSize": 17,
                      },
                      undefined,
                    ],
                  ]
                }
              >
                Correios
              </Text>
              <Text
                numberOfLines={1}
                style={
                  Array [
                    Object {
                      "color": "#000000",
                      "fontFamily": "System",
                      "fontWeight": "400",
                      "textAlign": "left",
                    },
                    Array [
                      Object {
                        "fontSize": 11,
                      },
                      Object {
                        "color": "rgba(255, 255, 255, 0.7)",
                      },
                      undefined,
                    ],
                  ]
                }
              >
                Rastreio de pacotes
              </Text>
            </View>
          </TouchableWithoutFeedback>
          <View
            style={
              Object {
                "width": 48,
              }
            }
          />
        </View>
      </SafeAreaView>
      <View
        style={
          Object {
            "backgroundColor": "#EAEBED",
            "flex": 1,
          }
        }
      >
        <View>
          <Text>
            iPad
             - 
            05/02/2020
          </Text>
          <Text>
            Apple Pencil
             - 
            07/02/2020
          </Text>
        </View>
      </View>
      <View
        pointerEvents="auto"
        style={
          Object {
            "backgroundColor": "#01417F",
            "borderRadius": 28,
            "bottom": 0,
            "elevation": 6,
            "margin": 16,
            "opacity": 1,
            "position": "absolute",
            "right": 0,
            "shadowColor": "#000000",
            "shadowOffset": Object {
              "height": 5,
              "width": 0,
            },
            "shadowOpacity": 0.24,
            "shadowRadius": 6,
            "transform": Array [
              Object {
                "scale": 1,
              },
            ],
          }
        }
      >
        <TouchableHighlight
          accessibilityComponentType="button"
          accessibilityRole="button"
          accessibilityStates={Array []}
          accessibilityTraits="button"
          activeOpacity={0.85}
          delayPressOut={100}
          disabled={true}
          style={
            Array [
              Object {
                "overflow": "hidden",
              },
              Object {
                "borderRadius": 28,
              },
            ]
          }
          underlayColor="rgba(255, 255, 255, 0.16)"
        >
          <View
            pointerEvents="none"
            style={
              Array [
                Object {
                  "alignItems": "center",
                  "flexDirection": "row",
                  "justifyContent": "center",
                },
                Object {
                  "height": 56,
                  "width": 56,
                },
              ]
            }
          >
            <View
              style={
                Array [
                  Object {
                    "alignItems": "center",
                    "justifyContent": "center",
                  },
                  Object {
                    "height": 24,
                    "width": 24,
                  },
                ]
              }
            >
              <View
                style={
                  Object {
                    "bottom": 0,
                    "left": 0,
                    "opacity": 1,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                    "transform": Array [
                      Object {
                        "rotate": "0deg",
                      },
                    ],
                  }
                }
              >
                <Text
                  accessibilityElementsHidden={true}
                  allowFontScaling={false}
                  importantForAccessibility="no-hide-descendants"
                  pointerEvents="none"
                  style={
                    Array [
                      Object {
                        "color": "#ffffff",
                        "fontSize": 24,
                      },
                      Array [
                        Object {
                          "transform": Array [
                            Object {
                              "scaleX": 1,
                            },
                          ],
                        },
                        Object {
                          "backgroundColor": "transparent",
                        },
                      ],
                      Object {
                        "fontFamily": "Material Design Icons",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                      Object {},
                    ]
                  }
                >
                  Ôêï
                </Text>
              </View>
            </View>
          </View>
        </TouchableHighlight>
      </View>
    </SafeAreaView>
  </View>
</View>
`;
